src/init.cpp

```diff
@@ -2096,7 +2096,22 @@ bool AppInitMain()
         ::feeEstimator.Read(est_filein);
     fFeeEstimatesInitialized = true;
 
-    // ********************************************************* Step 8: load wallet
+    // ********************************************************* Step 8a: load powcache.dat
```

Comment:
When the wallet was loaded (now in step 8b), it might trigger a rescan.  We want to have powcache loaded before that rescan occurs.